### ISSUE TITLE
the defined criterionMSE is not used

### ISSUE THREAD

**Issue by micklexqg** [BoyuanJiang/context_encoder_pytorch#10]
in your code, I find that the defined criterionMSE is not used.  the errD (BCE Loss)  is computed by  errD_fake = criterion(output, label). but for the errG_l2 (MSE loss) you compute it  by yourself (errG_l2 = (fake - real_center).pow(2) ) and not the defined criterionMSE.
and I find in lua version, the author used built-in loss function in torch for the MSE loss.
does it influence the results if not use the MSELoss function  in pytorch?
